src/util/sss_ptr_hash.c

```diff
@@ -332,7 +332,7 @@ void sss_ptr_hash_delete_all(hash_table_t *table,
 {
     struct sss_ptr_hash_value *value;
     hash_value_t *values;
-    size_t count;
+    unsigned long int count;
     size_t i;
     int hret;
```

Comment:
I think you should also change type of `i` to `unsigned long`.
And I would prefer just `unsigned long` instead of `unsigned long int`